components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -74,7 +76,8 @@ interface AwesomeBar {
         val chips: List<Chip> = emptyList(),
         val flags: Set<Flag> = emptySet(),
         val onSuggestionClicked: (() -> Unit)? = null,
-        val onChipClicked: ((Chip) -> Unit)? = null
+        val onChipClicked: ((Chip) -> Unit)? = null,
+        val weight: Int = Weight.DEFAULT
```

Comment:
I'd rename this to something like `score`. A provider overall might have a weight, which would use to "weight" its suggestions' scores (once equalized) against other providers. See my other comment.

Reply:
Sounds good!